app/lib/service/openid/github_openid.dart

```diff
@@ -55,7 +55,7 @@ class GitHubJwtPayload {
   /// the commit hash
   final String? sha;
 
-  /// RunId of the Github Action that this token comes from.
+  /// RunId of the GitHub Action that this token comes from.
```

Comment:
I'm limiting fixes to `GitHub` to comments / user facing strings which seems like a reasonable compromise. -- But I can drop this change entirely...

Reply:
Yeah, I think making the PR larger is a bad idea :rofl: 

If we want to try to do this to more than comments and user facing strings, it might be wiser to do it on one folder at the time.